### PR TITLE
Retire clean_metadata.py

### PR DESCRIPTION
**Pull request by krzyzacy**
https://github.com/kubernetes/test-infra/issues/4368

(in order to run clean_metadata.py)

/assign @rmmh @spxtr

### CODE REVIEW

File: images/gcloud/Dockerfile [hunk 1]
```diff
@@ -18,9 +18,12 @@ MAINTAINER senlu@google.com
 
 RUN apt-get update && apt-get install -y \
     python \
+    python-pip \
     wget && \
     apt-get clean
 
+RUN pip install requests
```
Comment (dims) [inline images/gcloud/Dockerfile]: Are we ok downloading latest version of requests as of the time the image is built? (or do we want to pin specific versions of requests)?

Comment (krzyzacy) [reply images/gcloud/Dockerfile]: seems other places like https://github.com/kubernetes/test-infra/blob/b6bb342cb0585ccf207aa65718990c628d4f9386/verify/test_requirements.txt#L3 does not pin version for requests. And we'd probably convert all those to bazel rules in the near future.

**Comment by krzyzacy** [comment]
(A better fix is that we can just retire this script - pr jobs runs completely fine without that for a month :crossed_swords: )
/assign @fejta

**Pull request merged**